Cargo.toml

```diff
@@ -45,6 +45,12 @@ url = { version = "2.2.2", optional = true }
 ttrpc = { version = "0.7.1", features = [ "async" ], optional = true }
 walkdir = "2"
 zstd = "0.11"
+fuse-backend-rs = "0.10.1"
+lazy_static = "1"
+
+nydus-api = "0.2.1"
+nydus-app = "0.3.2"
```

Comment:
It would be better to avoid dependency on `nydus-app`, we may provide a fake BuildTimeInfo for image-rs.

Reply:
Got it.